homeassistant/components/somfy/__init__.py

```diff
@@ -0,0 +1,162 @@
+"""
+Support for Somfy hubs.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/somfy/
+"""
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant import config_entries
+from homeassistant.components.somfy import config_flow
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_TOKEN
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.typing import HomeAssistantType
+from homeassistant.util import Throttle
+
+API = 'api'
+
+DEVICES = 'devices'
+
+REQUIREMENTS = ['pymfy==0.5.1']
```

Comment:
This can be removed as a result of adding the manifest.json file.

Reply:
```suggestion
```